platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiToolbarContainer/Style.less

```diff
@@ -129,10 +129,10 @@
 
 	.NormalContainerBlock .ToolbarContainer {
 		background: none;
-		height: 41px;
+		height: @navbarHeight; // 41px;

```

Comment:
Please remove the comment (this remark is valid for all others similar changes), it is not really useful and hard to keep up to date with the real default value IMO.